jsx/Panel.js

```diff
@@ -65,7 +76,11 @@ Panel.propTypes = {
   title: React.PropTypes.string
 };
 Panel.defaultProps = {
+  initCollapsed: false,
   id: 'default-panel',
-  height: '100%',
-  title: 'Selection Filter'
+  height: '100%'
 };
+
+window.Panel = Panel;
```

Comment:
@driusan Jake added this to all JS files when converting "transpilation" to webpack, in order to have backwards compatibility. 
Essentially, all our files are now ES6 modules that have to be included using an `import` statement. Assigning it to window allows to required this file a regular JS file as a fallback option. Since AFAIK Panel is used only through imports, it can probably be removed.